tests/out/hlsl/access.hlsl

```diff
@@ -24,7 +24,7 @@ float4 foo(uint vi : SV_VertexID) : SV_Position
     float4x4 matrix_ = float4x4(asfloat(bar.Load4(0+0)), asfloat(bar.Load4(0+16)), asfloat(bar.Load4(0+32)), asfloat(bar.Load4(0+48)));
     uint2 arr[2] = {asuint(bar.Load2(104+0)), asuint(bar.Load2(104+8))};
     float b = asfloat(bar.Load(0+48+0));
-    int a = asint(bar.Load((((NagaBufferLengthRW(bar) - 120) / 8) - 2u)*8+120));
+    int a = asint(bar.Load((((NagaBufferLengthRW(bar) - 120) / 4) - 2u)*4+120));
```

Comment:
Was this supposed to change? Should we have updated `tests/in/access.wgsl` to use a dummy struct?

Reply:
good idea!